docs/HowToCertificatesRotation.md

```diff
@@ -0,0 +1,473 @@
+# Rotating NuoDB TLS Certificates
+
+<details>
+<summary>Table of Contents</summary>
+<!-- TOC -->
+
+- [Rotating NuoDB TLS Certificates](#rotating-nuodb-tls-certificates)
+    - [Introduction](#introduction)
+        - [Terminology](#terminology)
+    - [About Key Rotation](#about-key-rotation)
+        - [Update Key Pair Certificates](#update-key-pair-certificates)
+            - [Using Kubernetes Secrets](#using-kubernetes-secrets)
+            - [Using HashiCorp Vault](#using-hashicorp-vault)
+        - [Verify Domain Certificates](#verify-domain-certificates)
+    - [NuoDB TLS Key Rotation](#nuodb-tls-key-rotation)
+        - [Rotate CA Certificate](#rotate-ca-certificate)
+        - [Rotate Server Certificate](#rotate-server-certificate)
+        - [Rotate Client Certificate](#rotate-client-certificate)
+        - [Cleanup](#cleanup)
+
+<!-- /TOC -->
+</details>
+
+## Introduction
+
+When using TLS encryption, it is necessary to rotate key pair certificates before the certificates expire.
```

Comment:
It is a customers decision, but should we recommend some base expiry times & review points?  All the examples use validity 36500

Reply:
Good point. We should probably use 365 in the examples to be more realistic.

Reply:
I totally agree. Maybe I copied this from the `setup-keys.sh` script.  ;)